src/rules/remove-empty-list-markers.ts

```diff
@@ -22,7 +22,7 @@ export default class RemoveEmptyListMarkers extends RuleBuilder<RemoveEmptyListM
   }
   apply(text: string, options: RemoveEmptyListMarkersOptions): string {
     return ignoreListOfTypes([IgnoreTypes.code, IgnoreTypes.yaml, IgnoreTypes.link, IgnoreTypes.wikiLink, IgnoreTypes.tag], text, (text) => {
-      const emptyListMarkerRegex = /^\s*(>\s*)*(-|\*|\+|\d+.|- (\[( |x)\]))\s*?$/gm;
+      const emptyListMarkerRegex = /^\s*(>\s*)*(-|\*|\+|\d+\D|- (\[( |x)\]))\s*?$/gm;
```

Comment:
My sole concern is that this should be `\.`. I guess I messed that before.

Reply:
We could also allow `)` as it is supported by [CommonMark](https://commonmark.org/help/)

Reply:
I am fine with that as well. It would then just need an or in the regex to check either `.` or `)`. So the following should do the trick: `(\.|\))`.